common/src/main/java/xyz/wagyourtail/jsmacros/client/api/library/impl/FWorld.java

```diff
@@ -137,154 +178,6 @@ public BlockDataHelper getBlock(BlockPosHelper pos) {
         return getBlock(pos.getX(), pos.getY(), pos.getZ());
     }
 
-    /**
-     * @since 1.6.4
-     * @param id
-     * @param chunkrange
-     *
-     * @return
-     */
-    public List<PositionCommon.Pos3D> findBlocksMatching(int centerX, int centerZ, String id, int chunkrange) {
-        return findBlocksMatchingInternal(centerX, centerZ, s -> Registry.BLOCK.getId(s).toString().equals(id), null, chunkrange);
-    }
-
-    /**
-     * @since 1.6.4
-     * @param id
-     * @param chunkrange
-     *
-     * @return
-     */
-    public List<PositionCommon.Pos3D> findBlocksMatching(String id, int chunkrange) {
-        assert mc.player != null;
-        int playerChunkX = mc.player.getBlockX() >> 4;
-        int playerChunkZ = mc.player.getBlockZ() >> 4;
-        return findBlocksMatchingInternal(playerChunkX, playerChunkZ, s -> Registry.BLOCK.getId(s).toString().equals(id), null, chunkrange);
-    }
-
-
-    /**
-     * @since 1.6.4
-     * @param ids
-     * @param chunkrange
-     *
-     * @return
-     */
-    public List<PositionCommon.Pos3D> findBlocksMatching(List<String> ids, int chunkrange) {
-        assert mc.player != null;
```

Comment:
would be better to mark these `@Deprecated` and have them use WorldScanner internally